config/webpack.config.dev.js

```diff
@@ -74,7 +74,7 @@ module.exports = merge({
         loader: 'string-replace',
         query: {
           search: '@@API',
-          replace: 'http://localhost:8000/data-research/consumer-complaints/api/v1/'
```

Comment:
um... how is this working for you?

I am running `cfgov-refresh` and that path is where the API is running.  What does your setup look like?